hypothesis-python/setup.py

```diff
@@ -117,7 +117,9 @@ def local_file(name):
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
+        "Topic :: Education :: Testing",
```

Comment:
I don't really understand that the intended meaning of this Trove classifier is. It seems reasonable enough based on the other things with this classifier though.

What do people even use trove classifiers for?

Reply:
Package discovery, mostly - if you find a package you like, click the classifier and scroll through the list of other packages with that tag.  This is probably most useful for the framework classifiers. 

Plus a bit of "declaration of intent", regarding development status, python versions (which I think only exist because ~~tags~~ classifiers predate the python_requires metadata), etc.

So as far as I know there is no generally intended meaning, just "this tag seems relevant" to aid discovery.